test/fancylogger.py

```diff
@@ -75,6 +75,8 @@ class FancyLoggerTest(TestCase):
     fd = None
 
     def setUp(self):
+        super(FancyLoggerTest, self).setUp()
```

Comment:
in the sense that it doesn't matter, it has the correct side effects. i'm still ruling out any possible causes.